frontend/src/app/components/dashboard/db-table-view/db-table-row-view/db-table-row-view.component.html

```
@@ -1,6 +1,10 @@
+<div *ngIf="selectedRow"
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)